pkg/artifactory/resource/security/resource_artifactory_group_test.go

```diff
@@ -122,8 +124,16 @@ func TestAccGroup_full(t *testing.T) {
 
 	configs := []string{}
 	for step, template := range templates {
-		configs = append(configs, acctest.ExecuteTemplate(fmt.Sprint(step), template, map[string]string{"groupName": groupName}))
-
+		configs = append(
+			configs,
+			acctest.ExecuteTemplate(
+				fmt.Sprint(step),
+				template, map[string]string{
+					"groupName": groupName,
```

Comment:
I am know I am being a PITA, but can `map` be on the next line 🙄

Reply:
😄 that's my bad. I missed this one.